sdk/nodejs/runtime/closure/codePaths.ts

```diff
@@ -79,8 +79,13 @@ export async function computeCodePaths(
 
 function isSubsumedByHigherPath(path: string, pathSet: Set<string>): boolean {
     for (const otherPath of pathSet) {
-        if (path.length > otherPath.length && path.startsWith(otherPath)) {
-            return true;
+        if (path.length > otherPath.length &&
+            path.startsWith(otherPath)) {
+
+            // Have to make sure we're actually a sub-directory of that other path.  For example,
+            // if we have:  node_modules/mime-types, that's not subsumed by node_modules/mime
+            const nextChar = path.charAt(otherPath.length);
+            return nextChar === "/" || nextChar === "\\";
```

Comment:
Should we use proper `path` operations here - breaking the part up into parts and verifying that one list is a subset of the other?

Reply:
@lukehoban I thought about it (also based on a stackoverflow post i saw on the subject) :)  However, i couldn't think of any thing that woudl catch that this would not be correct for.

That said... if we encounter any more issues here, it's likely we should scrap this approach.  However, it feels like it would be a bunch of unnecessary complexity now given that the above seem to work and address the only issue we've discovered here.